### ISSUE TITLE
#4. Research into what type of model we wish to attempt to implement first.

### ISSUE THREAD

**Issue by loketi** [Quantyra/kaggle-icr-challenge#4]

**Comment by babasahebpinjar** [comment]
This is multi-class classification problem.
XGBoost and random forest are used but the performance is not good. We can consider XGboost as a baseline model and improve it or try Neural networks or SVM .

**Comment by dfredriksen** [comment]
@babasahebpinjar This is not necessarily just a multi-class classification problem - the patient has none of the three age-related diseases or has any one of the three (Class A binary target: 1 indicates the subject has been diagnosed with one of the three conditions, 0 indicates they have not.). That being said, we can always use an ensemble model that predicts the odds of each class.

Also -- the submission file needs to have the prediction of the odds of the classes and not necessarily a binary classification result. In the case of a logistic regression model, you would be outputting the log odds and not the class label.

I would look beyond the standard ML models here and at some research papers. One type of model that I have seen used in life sciences is a model called 'conformal prediction', which provides a probability/prediction interval for the odds of something being part of a classification set (https://arxiv.org/abs/2306.09335).

Finding other such models and linking to the respective research papers here would be an excellent start to this ticket. Then we can experiment with all the approaches and see which gives us good results. Check out https://arxiv.org.
